hoptimator-kafka-adapter/src/main/java/com/linkedin/hoptimator/operator/kafka/KafkaTopicAclReconciler.java

```diff
@@ -0,0 +1,134 @@
+package com.linkedin.hoptimator.operator.kafka;
+
+import com.linkedin.hoptimator.operator.Operator;
+import com.linkedin.hoptimator.operator.ConfigAssembler;
+import com.linkedin.hoptimator.models.V1alpha1KafkaTopic;
+import com.linkedin.hoptimator.models.V1alpha1Acl;
+import com.linkedin.hoptimator.models.V1alpha1AclSpec;
+
+import io.kubernetes.client.extended.controller.reconciler.Reconciler;
+import io.kubernetes.client.extended.controller.reconciler.Request;
+import io.kubernetes.client.extended.controller.reconciler.Result;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1OwnerReference;
+
+import org.apache.kafka.common.acl.AccessControlEntry;
+import org.apache.kafka.common.acl.AclBinding;
+import org.apache.kafka.common.acl.AclOperation;
+import org.apache.kafka.common.acl.AclPermissionType;
+import org.apache.kafka.common.resource.PatternType;
+import org.apache.kafka.common.resource.ResourcePattern;
+import org.apache.kafka.common.resource.ResourceType;
+import org.apache.kafka.clients.admin.AdminClient;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+public class KafkaTopicAclReconciler implements Reconciler {
+  private final static Logger log = LoggerFactory.getLogger(KafkaTopicAclReconciler.class);
+  private final static String ACL = "hoptimator.linkedin.com/v1alpha1/Acl";
+  private final static String KAFKATOPIC = "hoptimator.linkedin.com/v1alpha1/KafkaTopic";
+
+  private final Operator operator;
+
+  public KafkaTopicAclReconciler(Operator operator) {
+    this.operator = operator;
+  }
+
+  @Override
+  public Result reconcile(Request request) {
+    log.info("Reconciling request {}", request);
+    String name = request.getName();
+    String namespace = request.getNamespace();
+
+    try {
+      V1alpha1Acl object = operator.<V1alpha1Acl>fetch(ACL, namespace, name);
+
+      if (object == null) {
+        log.info("Object {}/{} deleted. Skipping.", namespace, name);
+        return new Result(false);
+      }
+
+      String targetKind = object.getSpec().getResource().getKind();
+
+      if (!targetKind.equals("KafkaTopic")) {
+        log.info("Not a KafkaTopic Acl. Skipping.");
+      }
+
+      V1alpha1AclSpec.MethodEnum method = object.getSpec().getMethod();
+      AclOperation operation;
+      switch (method) {
+      case READ:
+        operation = AclOperation.READ;
+        break;
+      case WRITE:
+        operation = AclOperation.WRITE;
+        break;
+      default:
+        log.info("Unsupported KafkaTopic Acl operation {}. Skipping.", method);
```

Comment:
is this actually `info` or `warn` or `error` something?

Reply:
This would be normal, e.g. if some other reconciler handled this request instead of this one. I suppose it could be WARN.